scripts/check-cla

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I don't know how to verify this in the GitHub review UI, but make sure that this file is marked as Executable in git.

Reply:
Also, consider giving an explicit `.sh` extension to this file and `check-lint`.